docs/src/documentation/06-kiwi-use/02-content/04-specific-areas/social-media/02-visual-treatment.mdx

```diff
@@ -14,12 +15,15 @@ Our photography should always have a sense of optimism.
 The subjects and environments should feel authentic.
 We shouldn't be afraid of showing **real faces and emotions.**
 People should be **unposed** and their **composition** within the environment **natural**.
-Lighting, color and tone should be **unfiltered and realistic \#NoFilter!**
+Lighting, color, and tone should be **unfiltered and realistic \#NoFilter!**
 
 Each image should establish an emotional connection with our audience,
 showing them the world Kiwi.com can offer.
 
+<!-- prettier-ignore-start -->
+<!-- eslint-disable-next-line -->
 ## Our destinations...
+<!-- prettier-ignore-end -->
```

Comment:
There are several more instances of this:
```suggestion
<!-- eslint-disable -->
## Our destinations...
<!-- eslint-enable -->
```

Reply:
I know I fixed that at some point. Must have rebased my way out of it. 😞